.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: java
+jdk:
+  - oraclejdk7
+  - openjdk7
+
+before_install:
+ - chmod +x gradlew
```

Comment:
Don't let a windows user touch your repo :P

I'd like the build  to fail it anybody changes the execution permission, they would make my life harder